### PR TITLE
Pass args to iml-diagnostics

### PR DESCRIPTION
**Pull request by jgrund**
pass args from argv.

In addition, add some more copying of interesting files from the agent.

### CODE REVIEW

File: iml_sos_plugin/cli.py [hunk 1]
```diff
@@ -12,6 +12,8 @@ def flat_map(fn, xs):
 
 
 def main():
+    args = sys.argv[1:]
```
Comment (petertix) [inline iml_sos_plugin/cli.py]: Any check for valid args?

Comment (jgrund) [reply iml_sos_plugin/cli.py]: It's being forwarded to another cli whose job would be to do that.

Comment (petertix) [reply iml_sos_plugin/cli.py]: Sounds good.

Comment (petertix) [reply iml_sos_plugin/cli.py]: I see that you set:
  requires_root = True

in iml.py

**Pull request merged**